zmon_worker_monitor/builtins/plugins/cassandra_wrapper.py

```diff
@@ -20,6 +21,10 @@ def configure(self, conf):
         Called after plugin is loaded to pass the [configuration] section in their plugin info file
         :param conf: configuration dictionary
         """
+
+        self._username = conf.get('username', conf.get('user', None))
```

Comment:
We should use "user" and "pass" for consistency, because that's what the other plugins use (MySQL, PostgreSQL, ..)